monitor/README.md

```diff
@@ -0,0 +1,133 @@
+SeaClouds Monitor
+==================
+
+The SeaClouds monitor module is responsible, as its name suggests, to monitor SeaClouds applications at runtime. To this aim we mainly rely on Tower4Clouds (ref) a monitoring platform for multi-clouds application developed in the context of the MODAClouds FP7 European Project (ref). The released platform has been extended and customized in order satisfy specific SeaClouds monitoring requirements. More specifically, since Tower4Clouds relies on the concept of Data Collector (DC), or the components responsible to collect monitoring metrics, as main the mechanism to extend the Platform, a number of new Tower4Clouds's Data Collectors has been developed in SeaCloud. Moreover, being Tower4Clouds already integrated with some suitable external tools enabling visualization of monitoring data, like InfluxDB and Graphite, we decided to exploit this integration as a way to provide application metrics visualization in SeaClouds.
+Overall the SeaClouds Monitor architecture is depitched in the following figure:
+
+![alt tag](seaclouds-monitor.png "Optional title")
+
```

Comment:
nice picture, is there a particular deployment order on those components/subcomponents?
they can be deployed independently and in parallel?